docs/02-deployment-guide.md

```diff
@@ -391,6 +391,27 @@ comma separated list of stack names, for example:
 sleeper.optional.stacks=CompactionStack,IngestStack,QueryStack
 ```
 
+### Utility Scripts
+There are scripts in the `scripts/deploy` directory that can be used to manage an existing instance.
+
+#### Update Existing Instance
+The `deployExisting.sh` script can be used to bring an existing instance up to date with the version of Sleeper 
+in the EC2 environment.
```

Comment:
We can probably remove "with the version of Sleeper in the EC2 environment" as it explains what it does on the next line.